google/config.go

```diff
@@ -87,13 +92,19 @@ func (c *Config) loadAndValidate() error {
 		// your service account.
 		client = conf.Client(oauth2.NoContext)
 
+		tokenSource = conf.TokenSource(context.Background())
 	} else {
 		log.Printf("[INFO] Authenticating using DefaultClient")
 		err := error(nil)
 		client, err = google.DefaultClient(oauth2.NoContext, clientScopes...)
 		if err != nil {
 			return err
 		}
+
+		tokenSource, err = google.DefaultTokenSource(oauth2.NoContext, clientScopes...)
```

Comment:
oauth2.NoContext was deprecated and replaced by context.Background(); changing occurences in this file to context.Background().